CMakeLists.txt

```diff
@@ -429,27 +431,34 @@ if(WIN32)
         "${OpenMW_SOURCE_DIR}/Docs/license/DejaVu Font License.txt"
         "${OpenMW_BINARY_DIR}/settings-default.cfg"
         "${OpenMW_BINARY_DIR}/gamecontrollerdb.txt"
-        "${OpenMW_BINARY_DIR}/Release/openmw.exe"
         DESTINATION ".")
+	INSTALL(FILES "${OpenMW_BINARY_DIR}/Debug/openmw.exe" DESTINATION "." CONFIGURATIONS Debug)
+	INSTALL(FILES "${OpenMW_BINARY_DIR}/Release/openmw.exe" DESTINATION "." CONFIGURATIONS Release;RelWithDebInfo;MinSizeRel)
```

Comment:
I just pushed a commit that moves the install commands for the binaries to the target-specific CMakeLists.  This completely fixes the problem with the install target only working for release builds.  Now there are only a few configuration-specific install commands left in the main file, mostly dealing with DLL's.  I don't know of any way to eliminate those.